doc/conf.py

```diff
@@ -149,6 +149,10 @@
     ],
     }
 
+html_context = {
+    'css_files': ['_static/custom.css'],
```

Comment:
there is no `custom.css` in our repo and I don't see one added here, is this an errant commit?